backend/data/projects.js

```diff
@@ -340,6 +340,70 @@ const confirmEmailSent = async (projectId, error, info) => {
   }
 };
 
+
+const addEquipment = async (projectId, items) => {
+  // Get the project with the given projectID
+  projectCollection = await projects();
+  let currentProject = await projectCollection.findOne({_id: new ObjectId(projectId)});
+  if (!currentProject) {
+    throw new Error(`Project with ID ${projectId} not found`);
+  }
+
+  // Create equipmentRequired, deliveredEquipment, and neededEquipment arrays if they don't exist
+  if (!currentProject.equipmentRequired) {
```

Comment:
You can initialize the empty arrays for these fields when creating a project in createProject()